part1.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+public function postAction()
+{
+//avoid notices warnings
+    !isset($_POST['livechat_license_number']) ? $livechat_license_number = '' : $livechat_license_number =

```

Comment:
we should Use the $post["livechat_license_number"] instead of using the Globel variable $_POST